libbeat/docs/outputconfig.asciidoc

```diff
@@ -514,7 +514,7 @@ configured. The default value is 2.
 deprecated[5.0.0]
 
 The default port to use if the port number is not given in <<hosts,`hosts`>>. The default port number
-is 10200.
+is 5044.
```

Comment:
Whas this change included in cherry-picked PRs?

Reply:
@jsoriano Not sure exactly what you are asking here. I have backported this change to 6.3, 6.4, and 6.x. The change is relevant to earlier releases, too, but we don't usually backport all doc fixes to all releases where the docs are wrong because that would be really time consuming. Does that answer your question? Note that the default was changed here: https://github.com/elastic/beats/commit/eefdbd6f0a4ad64bf3208f7dc27a9805136c4bf3

Reply:
O, I see, yes, I meant that I couldn't find the change on the default port, it LGTM then, thanks!